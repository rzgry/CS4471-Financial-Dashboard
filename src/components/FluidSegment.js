import React from 'react';
import { Segment } from 'semantic-ui-react';

const FluidSegment = ({ children }) => <Segment style={{ height: '100%' }}>{children}</Segment>;

export default FluidSegment;
