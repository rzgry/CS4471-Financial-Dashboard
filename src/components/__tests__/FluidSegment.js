import React from 'react';
import renderer from 'react-test-renderer';
import FluidSegment from '../FluidSegment';

describe('<FluidSegment />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FluidSegment />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
