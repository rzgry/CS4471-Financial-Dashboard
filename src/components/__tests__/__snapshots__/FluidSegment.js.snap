// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FluidSegment /> renders correctly 1`] = `
<div
  className="ui segment"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
